changelog.d/12062.feature

```diff
@@ -0,0 +1 @@
+Add module callbacks to react to user deactivation status changes (i.e. deactivations and reactivations) and profile updates.
```

Comment:
Would it be worth linking to the documentation for these new methods here?

Reply:
It might be useful if module developers want to start using the new methods? We don't name them here either.